apps/site/config/prod.exs

```diff
@@ -51,7 +51,7 @@ config :site,
 config :logger,
   level: :info,
   handle_sasl_reports: true,
-  backends: [:console]
+  backends: [:console, Sentry.LoggerBackend]
```

Comment:
It's possible to do sourcemaps on the backend.
[docs](https://hexdocs.pm/sentry/Sentry.Sources.html).  
[config](https://github.com/mbta/glides/pull/431/files#diff-8e72c709985cb6a43cc0ab32cb487b8c17d2b2d003338ea42a4e2ad2829979dcR81).
[You also have to recompile your app an extra time](https://github.com/mbta/glides/pull/431/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R63).